### PR TITLE
refactor: Refactor bash scripts to Python

### PR DESCRIPTION
**Pull request by DerekMaggio**
# Overview

Refactor bash scripts with multiple clis, into a single python cli

# Changelog

- Remove old bash scripts
- Remove old README
- Add python cli for creating Virtual Machines and emulated systems
- Add Confluence documentation [here](https://opentrons.atlassian.net/wiki/spaces/RPDO/pages/1044414467/Opentrons+Emulation)

# Review requests

My goal is to make using emulation as simple as possible. 
Refer to the Confluence documentation listed above. 
If things are unclear then they need to be documented better. 
Please help me find these things

# Risk assessment

None. All Github Actions refer to the previous released version of the repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,248 +1 @@
...
-When that happens it can be emulated at the firmware level
-
-### Ubuntu Requirement
-
-For running either OT-3 a system running [Ubuntu](https://ubuntu.com/) is required.
-This is due to the usage of the [SocketCan](https://en.wikipedia.org/wiki/SocketCAN) library
-which is part of the Linux kernel.
-
-To meet this requirement, you can also create a virtual machine installed on [VirtualBox](https://www.virtualbox.org/)
+Refer to [Opentron Emulation Confluence Page](https://opentrons.atlassian.net/wiki/spaces/RPDO/pages/1044414467/Opentrons+Emulation)
```
Comment (sfoster1) [inline README.md:1]: this repo is publically available and the confluence isn't - let's maybe keep this readme around not necessarily fully but at least in part

File: emulation_system/.flake8 [hunk 2]
```diff
@@ -0,0 +1,26 @@
+[flake8]
+
+# set line-length for future black support
```
Comment (sfoster1) [inline emulation_system/.flake8:3]: future, or current?

**Review by sfoster1 — APPROVED** [review]
Love it! So much better than bash

File: emulation_system/.flake8 (hunk 2 above)
Comment (DerekMaggio) [reply emulation_system/.flake8]: That's what happens when I just do a straight copy paste from another one of our repos

Comment (DerekMaggio) [reply emulation_system/.flake8]: https://github.com/Opentrons/opentrons-emulation/pull/27/commits/f6e65cc942696f86c0079bb2c23656b41ed4da54

File: README.md (hunk 1 above)
Comment (DerekMaggio) [reply README.md:1]: I was thinking about trying to export a PDF of Confluence and post it in this repo. In some automated fashion would be nice.

Comment (DerekMaggio) [reply README.md:1]: Maybe something with this? 
https://github.com/atlassian-api/atlassian-python-api

**Review by amitlissack — APPROVED** [review]
Great stuff

File: README.md (hunk 1 above)
Comment (DerekMaggio) [reply README.md:1]: https://opentrons.atlassian.net/browse/RET-316

**Pull request merged**